scss/login.scss

```diff
@@ -0,0 +1,225 @@
+body.modal-open {
+  #main-outlet {
+    min-height: 100vh;
+  }
+}
+
+#discourse-modal .create-account .modal-body {
+  max-height: none !important;
+}
+
+.d-modal.forgot-password-modal,
+.d-modal.create-account,
+.d-modal.create-account-modal,
+.d-modal.login-modal {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1300;
```

Comment:
Do we have a comment that can help to understand why the 1300?

Reply:
Yeah this can be deleted, is from Discourse and the style is already there.